src/linter-jscs.js

```diff
@@ -97,6 +97,19 @@ export default class LinterJSCS {
         }
       });
     }));
+
+    this.subscriptions.add(atom.commands.add('atom-text-editor', {
+      'linter-jscs:fix-file': () => {
+        const textEditor = atom.workspace.getActiveTextEditor();
+
+        if (!textEditor || textEditor.isModified()) {
```

Comment:
Not a problem, the `eslint` fix only works on a file on disk, which is why the check is there for it :wink:.